src/App.js

```diff
@@ -69,8 +69,8 @@ export class App extends Component {
     document.querySelector('body').scrollTop = 0
     document.querySelector('#app').scrollTop = 0
 
-    const [first, ...rest] = args.location.pathname.slice(1) || 'home'
-    document.title = 'Ascend NTNU - ' + first.toUpperCase() + rest
+    //const [first, ...rest] = args.location.pathname.slice(1) || 'home'
+    //document.title = 'Ascend NTNU - ' + first.toUpperCase() + rest
```

Comment:
Doesn't this work? It only changes the title in the tab

Reply:
It doesn't work, thought to do that afterwards. Could just fix it in this pull request as well

Reply:
Is it possible to make it work? Seems like the problem is the `args` provided. It is not essential though.

Reply:
could pass in the wanted text as an argument

Reply:
Maybe just add an option in the this.switchPage(...., title) such that it can be provided manually